backend/migrations/README

```diff
@@ -0,0 +1,14 @@
+Single-database configuration for Flask.
```

Comment:
Could you please add an extension to the name of this file? I think that would help different tools to display it nicely

Reply:
Also I would say we need here two sections for the ductumentation:

## How to set up a database from scratch 
Starting from `git clone ...` of the repo, how to set up a working database on a local machine.

## How to create a new migration (version of the database)
For example if we need to add a table.

Reply:
And then mention this document in the main backend readme.